dialogue/dialogue_loader.gd

```diff
@@ -201,7 +201,7 @@ func dump_data (datafile, data):
 				f.store_string ("\"".insert (0, x[0].insert (0, "\n    <choice goto=\"")))
 				f.store_string ("</choice>".insert (0, String (x[1]).insert (0, ">")))
 			f.store_string ("\n  ")
-		if i.has ("t") and not (i.has ("i") or i.has ("x")):
+		if i.has ("t") and not (i.get ("i") or i.has ("x")):
```

Comment:
Seems like Godot Dictionary's `has` returned `false` before, when the value was `Null`, or something like that. Because of that, this check was failing and it was dumping all the texts as empty. I've replaced it here with `get` which in turn will:
- check if the key exists in the dictionary, returning it's value
- return `Null` (which is "falsy") if it doesn't exist